rclpy/src/rclpy/_rclpy.c

```diff
@@ -597,6 +613,7 @@ rclpy_expand_topic_name(PyObject * Py_UNUSED(self), PyObject * args)
  * provided as pymsg_type to send messages over the wire.
  *
  * A ValueError is raised (and NULL returned) when the topic name is invalid.
+ * Raises ValueError if the capsules are not the correct types
```

Comment:
Can you rephrase the sentence above to match this one as well?

Reply:
Done 451a072e372eecc46e07096f8fdcb2278051bd54